source/Server/Server.csproj

```diff
@@ -22,7 +22,7 @@
     <PackageReference Include="Octopus.Configuration" Version="2.0.1" />
     <PackageReference Include="Octopus.Data" Version="4.2.5" />
     <PackageReference Include="Octopus.Diagnostics" Version="1.3.0" />
-    <PackageReference Include="Octopus.Server.Extensibility" Version="8.0.0" />
+    <PackageReference Include="Octopus.Server.Extensibility" Version="8.1.1-beta0001" />
```

Comment:
Is this the one we plan to tag as a release version and then update this PR, or am I thinking of another library?

Reply:
Yeah, we have a release branch for Extensibility 8.x.x as well as 9.x.x so those will need to be updated first so that we can get the versions numbers off pre-release mode :)